app/services/concerns/xccdf_report/rules.rb

```diff
@@ -76,7 +76,7 @@ def new_rules
       def save_rules
         new_profiles = Profile.where(ref_id: profiles.keys)
         add_profiles_to_old_rules(rules_already_saved, new_profiles)
-        Rule.import(
+        Rule.import!(
```

Comment:
This bang makes the whole import fail if a validation fails for any record.

Reply:
i'm not very familiar w/ this code, where is the error caught?

Reply:
It's not caught. It's raised, and sidekiq (the job worker) handles it by re-enqueuing the job